packages/platforms/delta/arm64/tn48m-dn/tn4810m-dn/platform-config/r0/src/python/arm64_delta_tn4810m_dn_r0/__init__.py

```diff
@@ -37,8 +37,8 @@ def baseconfig(self):
         subprocess.call('echo "-2" > /sys/bus/i2c/devices/i2c-2/2-0074/idle_state', shell=True)
         subprocess.call('echo "-2" > /sys/bus/i2c/devices/i2c-2/2-0075/idle_state', shell=True)
 
-        # Insert prestera kernel module
-        self.insmod("prestera_sw.ko")
-        self.insmod("prestera_pci.ko")
+        # Insert Marvell prestera modules by only probing prestera_pci module
+        # Suggested by Taras Chornyi
+        subprocess.call('modprobe prestera_pci', shell=True)
```

Comment:
Why is `self.insmod("prestero_pci.ko")` not working anymore?

Reply:
> Why is `self.insmod("prestero_pci.ko")` not working anymore?

Hi Paul,

Because the name of prestera driver is changed from "prestera_sw.ko" to "prestera.ko", it will be better if we use modprobe to probe prestero_pci driver only, modprobe will automatically help us to insert the necessary drivers, like prestera.ko here.

For more details, you can refer to the discussion: https://github.com/dentproject/dentOS/pull/43#issuecomment-777704551

Reply:
I wonder if a method `modprobe` should be added to `packages/base/all/vendor-config-onl/src/`, so only `self.modprobe("prestero")` needs to be used.

Reply:
> I wonder if a method `modprobe` should be added to `packages/base/all/vendor-config-onl/src/`, so only `self.modprobe("prestero")` needs to be used.

Yes, the self.modprobe function can be implemented in file: "packages\base\all\vendor-config-onl\src\python\onl\platform\base.py" in the future if needed.